StreetPatch.Data/Entities/DTO/SignUpData.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StreetPatch.Data.Entities
```

Comment:
Please change the namespace to match the folder - `StreetPatch.Data.Entities.DTO`